cross/p7zip/digests

```diff
@@ -1,3 +1,2 @@
-p7zip_9.20.1_src_all.tar.bz2 SHA1 1cd567e043ee054bf08244ce15f32cb3258306b7
-p7zip_9.20.1_src_all.tar.bz2 SHA256 49557e7ffca08100f9fc687f4dfc5aea703ca207640c76d9dee7b66f03cb4782
-p7zip_9.20.1_src_all.tar.bz2 MD5 bd6caaea567dc0d995c990c5cc883c89
+p7zip_15.14.1_src_all.tar.bz2 SHA1 9b15a79f94230fab9b9d4f9f532c723117145c7a
+p7zip_15.14.1_src_all.tar.bz2 MD5 92cca093312b5a71a7be7dc7d1d32509
```

Comment:
sha256 checksum is not available on Sourceforge...
So whats the procedure? Am i suppose to generate a sha256? i only provided the 2 checksums that were available by the sourceforge developers, as i would think they 'own' that deliverable of creating the checksums. thoughts?